### DISCUSSION TITLE
Trying to deploy MoodNFT contract to Sepolia. ( Contract shows sufficient gas, but execution says insufficient)

### DISCUSSION THREAD

**Discussion by Glow-in-the-dark** [Cyfrin/foundry-full-course-cu#256] [Q&A] [answered]
I am able to deploy the contract on ANVIL chain using similar steps, however, i am not able to deploy to SEPOLIA.

This is my Error:
![Screenshot 2023-07-05 at 10 15 05 PM](https://github.com/Cyfrin/foundry-full-course-f23/assets/2751458/65a360c8-71dd-46f6-841e-3265c9174f0a)

It says estimated gas is 1.04 ETH, and on my Metamask wallet, i have 2ETH ( sepolia ETH)
<img width="327" alt="Screenshot 2023-07-05 at 10 16 36 PM" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/2751458/ee174185-9217-4a65-bbb3-923877b8ad40">

However, it still says insufficient funds.

Why tho ?

This is my github, for easier debugging:
https://github.com/Glow-in-the-dark/Solidity-playground/tree/main/EtherJS_Foundry/NFT

**Comment by shikhar229169** [comment, marked as answer]
Ensure that you are passing the private of your metamask wallet and not the private key of anvil. 
But if you have passed the correct key, then the error signifies that the txn is being reverted.
Consider uploading it on github and provide the link of repo, that will be helpful to debug it.

**Reply by Glow-in-the-dark** [reply]
ahhhh yes !!! forgot to switch. And yup. works perfectly now.

**Reply by ghost** [reply]
@Glow-in-the-dark don't forget to mark this answer as the solution.

**Reply by Glow-in-the-dark** [reply]
Right ! Thank so much and Marked <3
